website/docs/r/sentinel_data_connector_microsoft_cloud_app_security.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_data_connector_microsoft_cloud_app_security"
+description: |-
+  Manages a Microsoft Cloud App Security Data Connector.
+---
+
+# azurerm_sentinel_data_connector_microsoft_cloud_app_security
+
+Manages a Microsoft Cloud App Security Data Connector.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "west europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-workspace"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_sentinel_data_connector_microsoft_cloud_app_security" "example" {
+  name                       = "example"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+- `log_analytics_workspace_id` - (Required) The ID of the Log Analytics Workspace that this Microsoft Cloud App Security Data Connector resides in. Changing this forces a new Microsoft Cloud App Security Data Connector to be created.
+
+* `name` - (Required) The name which should be used for this Microsoft Cloud App Security Data Connector. Changing this forces a new Microsoft Cloud App Security Data Connector to be created.
+
+---
+
+- `alerts_enabled` - (Optional) Should the alerts be enabled? Defaults to `true`.
```

Comment:
can we update this to use a `*` rather than a `-` - matching the other resources:

```suggestion
* `alerts_enabled` - (Optional) Should the alerts be enabled? Defaults to `true`.
```